Backend/app/provider-platform/dynamic-offer-driver-app/Main/src-read-only/Storage/Queries/SearchReqLocation.hs

```diff
@@ -87,6 +91,8 @@ instance ToTType' Beam.SearchReqLocation Domain.Types.SearchReqLocation.SearchRe
         Beam.instructions = instructions,
         Beam.lat = lat,
         Beam.lon = lon,
+        Beam.merchantId = Kernel.Types.Id.getId <$> merchantId,
+        Beam.merchantOperatingCityId = Kernel.Types.Id.getId <$> merchantOperatingCityId,
```

Comment:
seems like this table does not used at all in both bap and bpp @hkmangla 
should we remove it from code?